### ISSUE TITLE
SoftKeyboard moves layout under AppBarLayout

### ISSUE THREAD

**Issue by arhi23** [henrytao-me/smooth-app-bar-layout#69]
Hi, 
I encountered a problem.
When I set windowSoftInputMode to "adjustResize|stateHidden" in my Activity, and focus edittext in NestedScrollView - AppBarLayout isn't scrolled, and parent of edittext is moved under AppBarLayout. But if I swipe on NestedScrollView and after that I focus edittext, than everything works fine and AppBarLayout is scrolled correctly. Is it a bug, or I'm doing something wrong here?

```
<?xml version="1.0" encoding="utf-8"?>
<android.support.design.widget.CoordinatorLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent">
    <me.henrytao.smoothappbarlayout.widget.NestedScrollView
        android:layout_width="match_parent"
        android:layout_height="wrap_content">
        <LinearLayout
            android:id="@+id/contentParent"
            android:layout_width="match_parent"
            android:layout_height="wrap_content"
            android:orientation="vertical">
            <View
                android:id="@+id/stub"
                android:layout_width="match_parent"
                android:layout_height="200dp" />
            <EditText
                android:layout_width="match_parent"
                android:layout_height="50dp"
                android:layout_marginTop="8dp"/>
            <EditText
                android:layout_width="match_parent"
                android:layout_height="50dp" />
            <EditText
                android:layout_width="match_parent"
                android:layout_height="50dp" />
            <EditText
                android:layout_width="match_parent"
                android:layout_height="50dp" />
        </LinearLayout>
    </me.henrytao.smoothappbarlayout.widget.NestedScrollView>
    <me.henrytao.smoothappbarlayout.SmoothAppBarLayout
        android:id="@+id/app_bar_layout"
        android:layout_width="match_parent"
        android:layout_height="200dp">
        <android.support.design.widget.CollapsingToolbarLayout
            android:id="@+id/collapsing_toolbar"
            android:layout_width="match_parent"
            android:layout_height="match_parent">
            <ImageView
                android:id="@+id/photo"
                android:layout_width="match_parent"
                android:layout_height="match_parent"
                android:scaleType="centerCrop" />
            <android.support.v7.widget.Toolbar
                android:id="@+id/toolbar"
                android:layout_width="match_parent"
                android:layout_height="?attr/actionBarSize"/>
        </android.support.design.widget.CollapsingToolbarLayout>
    </me.henrytao.smoothappbarlayout.SmoothAppBarLayout>
</android.support.design.widget.CoordinatorLayout>

```

**Comment by henrytao-me** [comment]
Hi @arhi23 

Let me check and get back to you soon.

**Comment by henrytao-me** [comment]
Hi @arhi23 

It's fixed in v1.0.2. Check it out. 

Cheers, 
Henry
